internal/ingress/metric/main.go

```diff
@@ -113,7 +114,12 @@ func (c *collector) Start() {
 	c.registry.MustRegister(c.ingressController)
 	c.registry.MustRegister(c.socket)
 
-	go c.nginxStatus.Start()
+	// the default nginx.conf does not contains
+	// a server section with the status port
+	go func() {
+		time.Sleep(10 * time.Second)
```

Comment:
IMO 10s is a bit too much, potentially we can loose stats for thousands of requests. Maybe `5s`? For dynamic configuration we sleep only 1s and it has been fine so far.